JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,12 +1,29 @@
-//
-//  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-// 과일 저장소 타입
+enum DefaulSetting {
+    static let stock = 10
+}
+
 class FruitStore {
+    private var fruitsInventory : [Fruit : Int] = [:]
+    
+    init() {
+        Fruit.allCases.forEach { fruitsInventory[$0] = DefaulSetting.stock }
+    }
     
+    func isEnoughStock(for juiceIngredient: [Fruit : Int]) throws {
+        for (fruit, requiredAmount) in juiceIngredient {
+            guard let fruitStoreStock = fruitsInventory[fruit], fruitStoreStock >= requiredAmount  else {
+                throw JuiceMakeError.outOfStock
+            }
+        }
+    }
+    func decreaseFruitStock(by juiceIngredient: [Fruit : Int]) {
+        for (fruit, requiredAmount) in juiceIngredient {
+            if var fruitStoreStock = fruitsInventory[fruit] {
+                fruitStoreStock -= requiredAmount
+                fruitsInventory[fruit] = fruitStoreStock
+            }
```

Comment:
`fruitStoreStock` 이란 단어가 모호하고, 여기에 뭔가 변경을 해주고 그걸 다시 `inventory`에 집어넣고 있어 개인적으론 코드 이해가 어려운 것 같아요 🤔 
```suggestion
            if let currentAmount = fruitsInventory[fruit] {
                fruitsInventory[fruit] = currentAmount - requiredAmount
            }
```
현재 재고라는 것을 명확하게 명시하고, 주스를 만드는 데 필요한 양을 `requiredAmount`로 선언해주셨으니 같은 맥락으로 `amount`로 표현해주고 코드도 좀 더 간결하게 선언하는 게 좋을 것 같아요.

여기서 의미를 한 번 더 부여해주자면,
```suggestion
            if let currentAmount = fruitsInventory[fruit] {
                let remainingAmount = currentAmount - requiredAmount
                fruitsInventory[fruit] = remainingAmount
            }
```
요렇게도 가능할 것 같구요 🙂 
가독성에 대한 건 개인적인 의견이니 참고만 해주세요 🙏

Reply:
두번째로 제안해 주신 의견이 확실히 이전 코드보다 가독성이 좋아서 적용해보았습니다!!